frontend/src/lib/api/reaction.ts

```diff
@@ -20,14 +18,19 @@ type Reaction = decodeType<typeof reactionDecoder>;
 const ReactionAPI = {
     get: async (slug: string): Promise<Reaction | ErrorMessage> => {
         try {
-            const { data, status } = await axios.get(`${BACKEND_URL}/reaction/${slug}`, {
```

Comment:
her kan du bare gønne:
```typescript
 headers: {
      Authorization: `Bearer ${idToken}`,
  }
```

og bare sende inn `idToken` som argument i `ReactionAPI.get()`, som du får ved `useSession()`.

Altså istedenfor å gå gjennom Next sin `/api` for å få tak i `idToken`.

Reply:
Yes, er akkurat det jeg holder på med nå. Er ferdig snart.